common/src/test/java/com/sequenceiq/cloudbreak/validation/ChoiceValidatorTest.java

```diff
@@ -8,16 +8,16 @@
 import javax.validation.ValidatorFactory;
 
 import org.junit.Assert;
```

Comment:
junit4 assert

Reply:
Oh. I have done a search and replace for static imports, but not for non-static ones. It is quite error-prone without having  proper checkstyle support...